pyqtgraph/graphicsItems/DateAxisItem.py

```diff
@@ -29,46 +29,65 @@ def utcfromtimestamp(timestamp):
 MAX_REGULAR_TIMESTAMP = (datetime(9999, 1, 1) - datetime(1970,1,1)).total_seconds()
 SEC_PER_YEAR = 365.25*24*3600
 
+
+# The stepper functions provide
+#   'first' == True: The first tick value for 'val' being the minimum of the current view.
+#   'first' == False: The next tick value for 'val' being the previous tick value.
```

Comment:
tickey-tack comment but should probably change the comment to reflect first is True vs. `'first' == True`

Reply:
eh, on second thought, ignore this....